aws/resource_aws_db_instance.go

```diff
@@ -38,7 +38,7 @@ func resourceAwsDbInstance() *schema.Resource {
 		Timeouts: &schema.ResourceTimeout{
 			Create: schema.DefaultTimeout(40 * time.Minute),
 			Update: schema.DefaultTimeout(80 * time.Minute),
-			Delete: schema.DefaultTimeout(40 * time.Minute),
+			Delete: schema.DefaultTimeout(60 * time.Minute),
```

Comment:
Please note that the timeouts documentation needs to be updated in `website/docs/r/db_instance.html.markdown` for this change. 👍  Running acceptance testing now.

Reply:
Fixed